packages/clerk-js/src/ui/Components.tsx

```diff
@@ -297,6 +322,23 @@ const Components = (props: ComponentsProps) => {
     </LazyModalRenderer>
   );
 
+  const mountedUserVerificationModal = (
+    <LazyModalRenderer
+      globalAppearance={state.appearance}
+      appearanceKey={'userVerification'}
+      componentAppearance={userProfileModal?.appearance}
```

Comment:
Are we just reusing the `componentAppearance={userProfileModal?.appearance}` here for ease or is this supposed to be `componentAppearance={userVerificationModal?.appearance}`

Reply:
I don't think this is intentional, nice catch